### PR TITLE
Update description of get_labels

### PR DESCRIPTION
**Pull request by rimms**
This patch is a part of https://github.com/jubatus/jubatus_core/pull/272

I modified the description of API  to follow the new RPC signature.

### CODE REVIEW

File: source/en/api_classifier.rst [hunk 1]
```diff
@@ -226,11 +226,11 @@ Methods
...
       This API is designed to accept bulk classification with list of :mpidl:type:`datum`.
 
-   .. mpidl:method:: list<string> get_labels()
+   .. mpidl:method:: map<string, int> get_labels()
 
-      :return:     list of all labels in the jubatus
+      :return:     Pairs of label and the number of trained data
 
-      Returns all label list.
+      Returns the number of trained data for each label.
```
Comment (kmaehashi) [inline source/en/api_classifier.rst]: The expression `number of trained data` is ambiguous. Need to mention the unlearner behavior?

**Comment by kmaehashi** [comment]
@rimms Reviewed.

**Comment by rimms** [comment]
Fixed.

**Comment by kmaehashi** [comment]
:+1: LGTM

**Pull request merged**